test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -1,230 +1,732 @@
-        Set-Location -Path "TestDrive:\"
-
-        #build semi-complex directory structure to test navigation within
-        $level1_0 = "Level1_0"
-        $level2_0 = "Level2_0"
-        $level2_1 = "Level2_1"
-        New-Item -Path $level1_0 -ItemType Directory > $null
-        New-Item -Path (Join-Path $level1_0 $level2_0) -ItemType Directory > $null
-        New-Item -Path (Join-Path $level1_0 $level2_1) -ItemType Directory > $null
     }
 
     AfterAll {
         #restore the previous location
         Set-Location -Path $restoreLocation
     }
-
+    
     BeforeEach {
         Set-Location -Path "TestDrive:\"
     }
 
-    It "Verify Get-Location and Set-Location" {
-        $currentLoc = Get-Location
-        Set-Location $level1_0
-        $level1Loc = Get-Location
-        Set-Location $level2_0
-        $level2Loc = Get-Location
-        $currentLoc.Path | Should Be (Join-Path "TestDrive:" "")
-        $level1Loc.Path | Should Be (Join-Path "TestDrive:" $level1_0)
-        $level2Loc.Path | Should Be (Join-Path (Join-Path "TestDrive:" $level1_0) $level2_0)
-    }
-
-    It "Verify Push-Location and Pop-Location" {
-        #push a bunch of locations
-        Push-Location
-        $push0 = Get-Location
-        Set-Location $level1_0
-        Push-Location
-        $push1 = Get-Location
-        Set-Location $level2_0
-        Push-Location
-        $push2 = Get-Location
-
-        #navigate back home to change path out of all pushed locations
-        Set-Location "TestDrive:\"
-
-        #pop locations off
-        Pop-Location
-        $pop0 = Get-Location
-        Pop-Location
-        $pop1 = Get-Location
-        Pop-Location
-        $pop2 = Get-Location
-
-        $pop0.Path | Should Be $push2.Path
-        $pop1.Path | Should Be $push1.Path
-        $pop2.Path | Should Be $push0.Path
-    }
-}
+    Context "Valdiate New-Item parameters for the FileSystem provider" {
+        AfterEach {
+            Remove-Item -Path * -Recurse -Force -ErrorAction SilentlyContinue
+        }
 
-Describe "Validate basic Content Cmdlets for the FileSystem provider" -Tags "CI" {
-    BeforeAll {
-        $testFile = "testFile.txt"
-        $restoreLocation = Get-Location
-        Set-Location "TestDrive:\"
-    }
+        It "Verify Directory + Whatif" {
+            New-Item -Path . -ItemType Directory -Name $testDir -WhatIf > $null
+            try {
+                Get-Item -Path $testDir -ErrorAction Stop 
+                throw "Expected exception not thrown"
+            }
+            catch { $_.FullyQualifiedErrorId | Should Be "PathNotFound,Microsoft.PowerShell.Commands.GetItemCommand" }
+        }
 
-    AfterAll {
-        #restore the previous location
-        Set-Location -Path $restoreLocation
+        It "Verify Directory + Confirm" {
+            $result = New-Item -Path . -ItemType Directory -Name $testDir -Confirm:$false
+            $result.Name | Should Be $testDir
+        }
+
+        It "Verify Directory + Force" {
```

Comment:
This isn't a true "-Force" Test. The test should verify that Force works in a situation where the command would otherwise fail. In this case, the simplest way of doing it is to do the command twice. The first time without -Force and the second time with -Force. If -Force works properly, no exception should be thrown.